app/views/internal/tags/show.html.erb

```diff
@@ -31,6 +31,15 @@
     <%= f.label :supported %>
     <%= f.check_box :supported %>
   </div>
+  <div>
+    <%= f.label :badge_id %>
+    <% badges_array = Badge.all.map { |badge| [badge.title, badge.id] } %>
```

Comment:
This could be refactored to `Badge.pluck(:title, :id)`

Reply:
Definitely. Great call.